<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDemo53_Orden</class>
 <widget class="QWidget" name="dlgDemo53_Orden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>656</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mantenimiento de Ordenes</string>
  </property>
  <widget class="QPushButton" name="btnNuevo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QTableView" name="tblOrden">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>961</width>
     <height>521</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblRegistros">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total de Registros:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTotal">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>80</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
